### PR TITLE
Added Webtorrent Global Status IPC channel

### PR DESCRIPTION
**Pull request by phoenix1796**
With reference to #4 ,
Added IPC channel in main process for global status requests and a demonstration in the UI(check browser console).

P.S. Hope this helps :)

### CODE REVIEW

**Comment by retrohacker** [comment]
Ah, this is something I was going to explore! Thanks for putting it in here :smile:

File: ui/index.html [hunk 1]
```diff
@@ -35,6 +35,13 @@
           .addEventListener('click', peerWeb.search.refresh)
       })
     </script>
+    <script>
+      var ipcRenderer = require('electron').ipcRenderer;
+      ipcRenderer.on('global-status-reply',(event,arg)=>{
+        console.log(arg);//Testing only
```
Comment (retrohacker) [inline ui/index.html:41]: This is super sexy!

I'll wire this up to a dashboard sometime this week :heart:

**Comment by retrohacker** [comment]
I'm going to merge this into a feature branch to work off of if that is okay.

**Comment by phoenix1796** [comment]
Glad you liked it , :smile: 
I was thinking if there could be a status bar which would show the current status at the footer of the UI.

P.S. Just a request but can you post a infohash for a (peer://)site that is being seeded , for testing purposes.

**Comment by retrohacker** [comment]
@phoenix1796 Let me set one up really quick. I've been using https://github.com/feross/bittorrent-tracker on local host to test.

**Comment by retrohacker** [comment]
@phoenix1796 peer://000260db64bc4d57afb5458639f2364d628b6e1b/index.html is live

I'll leave this running. I'm working on a markdown file for those in the alpha, it will be useful if you want to seed your own websites :smile:

**Comment by retrohacker** [comment]
Just got done with work for the day. Heading downtown to a Linux User Group. Created a feature branch with your work so I can work on the train. Will merge from here: https://github.com/retrohacker/peerweb/tree/webtorrent_stats

**Pull request merged**